Samples/PublicSamples/RecordingBot/src/RecordingBot.Model/Models/JoinCallBody.cs

```diff
@@ -1,39 +1,8 @@
-﻿// ***********************************************************************
-// Assembly         : RecordingBot.Model
-// Author           : JasonTheDeveloper
-// Created          : 09-07-2020
-//
-// Last Modified By : dannygar
-// Last Modified On : 09-07-2020
-// ***********************************************************************
-// <copyright file="JoinCallBody.cs" company="Microsoft">
-//     Copyright ©  2020
-// </copyright>
-// <summary></summary>
-// ***********************************************************************
 namespace RecordingBot.Model.Models
 {
-    /// <summary>
-    /// The join call body.
-    /// </summary>
     public class JoinCallBody
     {
-        /// <summary>
-        /// Gets or sets the Teams meeting join URL.
-        /// </summary>
-        /// <value>The join URL.</value>
         public string JoinURL { get; set; }
-
-        /// <summary>
```

Comment:
Not a Ghost Doc

Reply:
I agree. This was a useful comment. Has been readded with the latest push